configure.ac

```diff
@@ -103,7 +103,7 @@ AC_MSG_RESULT(no)
 dnl clock_gettime is in librt for glibc <2.17
 AC_SEARCH_LIBS(clock_gettime, rt)
 
-AC_CHECK_FUNC(strlcpy, found_strlcpy=yes, found_strlcpy=no)
+AC_CHECK_FUNCS(strlcpy, found_strlcpy=yes, found_strlcpy=no)
```

Comment:
+1 for this

https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Generic-Functions.html

HAVE_STRLCPY is what we want